src/v35.js

```diff
@@ -28,8 +45,6 @@ export const URL = '6ba7b811-9dad-11d1-80b4-00c04fd430c8';
 
 export default function (name, version, hashfunc) {
   function generateUUID(value, namespace, buf, offset) {
-    const off = (buf && offset) || 0;
-
     if (typeof value === 'string') value = stringToBytes(value);
     if (typeof namespace === 'string') namespace = uuidToBytes(namespace);
```

Comment:
Since we're at it: I'm wondering why our eslint rules don't strictly demand curly braces? I'd prefer that…

Reply:
Whups, just seeing this.    As I said in my other comment, I'd err on the side of using default `prettier` rules, and avoiding custom rules simply for aesthetics.  Not something I feel strongly about though.